airmozilla/main/templates/main/event.html

```diff
@@ -98,8 +98,8 @@
      <header class="entry-header">
       <h1 class="entry-title">
         {{ event.title }}
-      </h1>
       <a class="star event" data-id="{{ event.id }}" title="Save as a starred event"></a>
```

Comment:
This is `.entry-title a.star` which is here: https://github.com/bugzPDX/airmozilla/blob/1147657/airmozilla/starred/static/starred/css/star_event.css#L31